src/monitor/monitor.c

```diff
@@ -173,6 +173,9 @@ struct mt_ctx {
     /* For running unprivileged services */
     uid_t uid;
     gid_t gid;
+
+    /* Dynamic library load */
+    struct sss_nss_ops ops;
```

Comment:
Here and everywhere, I would use more telling variable name.
`nss_ops` at least.

Reply:
Actually, content of this struct is totally opaque in this module. I.e. it could (and probably) should be `void *`.
This doesn't seem neat...

Reply:
This change has been removed.